cli/cppcheckexecutor.cpp

```diff
@@ -183,10 +183,13 @@ int CppCheckExecutor::check(int argc, const char* const argv[])
         return EXIT_SUCCESS;
     }
 
+    settings.loadSummaries();
```

Comment:
No need to check `settings.buildDir` before calling this since it will do that by itself.